js/modules/k6/http/http.go

```diff
@@ -240,6 +348,12 @@ func (*HTTP) request(ctx context.Context, rt *goja.Runtime, state *common.State,
 		tags["status"] = strconv.Itoa(resp.Status)
 		tags["proto"] = resp.Proto
 
+		if tlsState := res.TLS; res.TLS != nil {
```

Comment:
`tlsState` is longer than `res.TLS`. As far as shorthands go, that's pretty lousy.